src/actions/files.js

```diff
@@ -46,8 +47,13 @@ export function uploadFile(channelId, rootId, fileFormData, formBoundary) {
             files = await Client4.uploadFile(fileFormData, formBoundary);
         } catch (error) {
             forceLogoutIfNecessary(error, dispatch);
-            dispatch(batchActions([
-                {type: FileTypes.UPLOAD_FILES_FAILURE, error},
+            dispatch(batchActions([{
+                type: FileTypes.UPLOAD_FILES_FAILURE,
+                clientIds: parseClientIdsFromFormData(fileFormData),
```

Comment:
Might be better to pass the client IDs in rather than parse them here but not a big deal 0/5